src/terminal/adapter/MouseInput.cpp

```diff
@@ -695,37 +625,42 @@ void MouseInput::UseAlternateScreenBuffer()
 // <none>
 // Return value:
 // <none>
-void MouseInput::UseMainScreenBuffer()
+void MouseInput::UseMainScreenBuffer() noexcept
 {
-    _fInAlternateBuffer = false;
+    _inAlternateBuffer = false;
 }
 
 // Routine Description:
-// - Returns true if we should translate the input event (uiButton, sScrollDelta)
+// - Returns true if we should translate the input event (button, sScrollDelta)
 //      into an alternate scroll event instead of the default scroll event,
 //      dependiong on if alternate scroll mode is enabled and we're in the alternate buffer.
 // Parameters:
-// - uiButton: The mouse event code of the input event
-// - sScrollDelta: The scroll wheel delta of the input event
+// - button: The mouse event code of the input event
+// - delta: The scroll wheel delta of the input event
 // Return value:
 // True iff the alternate buffer is active and alternate scroll mode is enabled and the event is a mouse wheel event.
-bool MouseInput::_ShouldSendAlternateScroll(_In_ unsigned int uiButton, _In_ short sScrollDelta) const
+bool MouseInput::_ShouldSendAlternateScroll(const unsigned int button, const short delta) const noexcept
 {
-    return _fInAlternateBuffer &&
-           _fAlternateScroll &&
-           (uiButton == WM_MOUSEWHEEL || uiButton == WM_MOUSEHWHEEL) && sScrollDelta != 0;
+    return _inAlternateBuffer &&
+           _alternateScroll &&
+           (button == WM_MOUSEWHEEL || button == WM_MOUSEHWHEEL) && delta != 0;
 }
 
 // Routine Description:
 // - Sends a sequence to the input coresponding to cursor up / down depending on the sScrollDelta.
 // Parameters:
-// - sScrollDelta: The scroll wheel delta of the input event
+// - delta: The scroll wheel delta of the input event
 // Return value:
 // True iff the input sequence was sent successfully.
-bool MouseInput::_SendAlternateScroll(_In_ short sScrollDelta) const
+bool MouseInput::_SendAlternateScroll(const short delta) const noexcept
 {
-    const wchar_t* const pwchSequence = sScrollDelta > 0 ? CURSOR_UP_SEQUENCE : CURSOR_DOWN_SEQUENCE;
-    _SendInputSequence(pwchSequence, CCH_CURSOR_SEQUENCES);
-
+    if (delta > 0)

```

Comment:
more mechanical translation would be to make it a `const wstring_view&`

Reply:
meh